src/resources/i18n/sq.json

```
@@ -178,7 +178,7 @@
     "referralfrom": "Referuar nga",
     "referralto": "Referohet tek",
     "newpatient": "Lloji i frekuentimit",
-    "newadmittance": "Pranim i ri",
+    "newattendance": "Pranim i ri",
```

When changing the meaning of a translation, we have to reset the translation for languages other than English

```suggestion
    "newattendance": "New Attendance",
```